src/Conf.ml

```diff
@@ -514,6 +515,23 @@ module Formatting = struct
     C.choice ~names ~all ~env ~doc ~allow_inline:true ~update:(fun conf x ->
         {conf with sequence_style= x} )
 
+  let structure_item_grouping =
+    let doc = "Line breaks between structure items." in
```

Comment:
Better to rename, perhaps `module-item-spacing` (thinking again, 'grouping' is probably an implementation detail). And maybe reword doc to `Spacing between items of structures and signatures.`